src/types/Worker.ts

```diff
@@ -25,7 +25,8 @@ export interface WorkerSetElement {
   numberOfWorkerElements: number;
 }
 
-export interface WorkerMessage<T extends WorkerData> {
+// export interface WorkerMessage<T extends WorkerData> {
+export interface WorkerMessage<T> {
```

Comment:
T shall really be an extension of the type WorkerData (JsonType). The data attribute shall be JSON (de-)serializable.

Reply:
That was the idea originally. But it caused problem with `ChargingStationInfo` as it rely on type that comes from a library (`Record` in `ChargingStationTemplate`) which cannot be changed (well they can but that would be a non commitable change)